### PR TITLE
Fixed broken lolex integration.

### LINKED ISSUE #323: process-nextick-args used incorrectly
[The whole point of the change in `process-nextick-args` was to enable fake timers to work with this module. The way this was supposed to be accomplished was by making the nextTick property a "live" binding.](https://github.com/calvinmetcalf/process-nextick-args/pull/10) In https://github.com/nodejs/readable-stream/commit/41a330137ce9cda5a95ef23d5454c70acb9c962a#diff-279988bef5e6fbf7ed82cae28ecc2d59L128 this was changed to keep a reference to a potentially stale function. I'm not sure how to go about changing the generated code to make this change, else I would just make a PR.

### PR DESCRIPTION
**Pull request by kolodny**
Closes #323

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "assert": "^1.4.0",
     "babel-polyfill": "^6.9.1",
     "buffer": "^4.9.0",
+    "lolex": "^2.3.2",
```
Comment (mcollina) [inline package.json]: why was this added

Comment (kolodny) [reply package.json]: Whoops, forgot to include the `lolex-fake-timers.js` file. I just added it now

**Comment by mcollina** [comment]
@nodejs/github-bot do you know why this Travis has not been triggered here?

**Comment by joyeecheung** [comment]
Somehow Travis failed to receive the notification again...https://travis-ci.org/nodejs/readable-stream/builds
